ruby/rna-transcription/rna_transcription.rb

```diff
@@ -0,0 +1,21 @@
+class Complement
+  CONVERSION = {
+    G: 'C',
+    C: 'G',
+    T: 'A',
+    A: 'U'
+  }
+
+  def self.of_dna(dna)
+    convert = ''
+    if dna != ''
```

Comment:
What do you think about having a guard clause here? https://www.thechrisoshow.com/2009/02/16/using-guard-clauses-in-your-ruby-code/

Reply:
hmm yes, makes sense. I've never used one before either. Good article 👍